tools/modulelint.py

```diff
@@ -45,8 +45,28 @@ def setUp(self):
     def testDockerFromBaseruntime(self):
         self.assertTrue(self.dp.check_baseruntime())
 
-    def testDockerRunMicrodnf(self):
-        self.assertTrue(self.dp.check_microdnf())
+    def testDockerNodocs(self):
+        self.start()
+        # This returns a list of packages defined in config.yaml for testing
+        # e.g. ["bash", "rpm", "memcached"] in case of memcached
+        pkgs = self.backend.getPackageList()
+        for pkg in pkgs:
+            docs = self.run("rpm -qad %s" % pkg).stdout
```

Comment:
@ignatenkobrain Personally, I don't see any advantage via to use python rpm binding inside container.